compose.yml

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+
+services:
+  site:
+    build: "."
+    image: "ozone-site-dev"
+    ports:
+      - 3000:3000
+    volumes:
+      - ".:/ozone-site"
+      # The below option is used to prevent overwriting of the docker image
+      # node_modules by the mounted volume's node_modules
+      # This would create an empty node_modules in the working directory of the local machine if not already present.
```

Comment:
```suggestion
      # The below option is used to prevent overwriting the node_modules directory in the docker image
      # with the node_modules present in the mounted volume.
      # This will create an empty node_modules directory in the working directory of the host if it is not already present.
```

Reply:
Addressed in the latest commit